Update overview.md

jeffrey-opdam opened the pull request:
Fixing url to configure image.

Thank you for your pull request!
By completing this pull request, you agree to the [Contributing License Agreement](https://github.com/ALM-Rangers/Roll-Up-Board-Widget-Extension/blob/master/.github/CLA.md).

Fixes # .

Changes proposed in this pull request:  
- 
- 
- 

@ALM-Rangers/Roll-Up-Board-Widget-Extension

jeffrey-opdam committed:
Update overview.md

Fixing url to configure image.

jeffrey-opdam committed:
Update WidgetRollUpBoard.ts

Links give a 404, team name is not needed anymore.

wpschaub commented on src/vsarVSTS-Roll-up-board/overview.md:
```diff
@@ -41,4 +41,4 @@ We need your feedback! Here are some ways to connect with us:
 
-Review the [list of features and resolved issues of latest tools and extensions](https://aka.ms/vsarreleases) for information on the latest releases.
\ No newline at end of file
+Review the [list of features and resolved issues of latest tools and extensions](https://aka.ms/vsarreleases) for information on the latest releases.
```
We will need to migrate the https://aka.ms/vsarreleases page, re-create a similar one on the alm DevOps rangers site, or remove the reference in the overview page. THe current page will expire when they deprecate our MSDN blog.

jeffrey-opdam replied:
I am removing the reference for now.

wpschaub reviewed (approved):
Just a minor comment on the overview page, which we need to discuss, It can be addressed in a future update.

jeffrey-opdam committed:
Removed the references to the releases page

Removed the references to the releases page, need to create a new one on our new website later.
